src/maks.kulykov/DatabaseManager.java

```diff
@@ -0,0 +1,48 @@
+package maks.kulykov;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DatabaseManager {
+    private String db = "mydb";
+    private String userName = "postgres";
+    private String password = "postgres";
+
+    private String cmdDB;
+    private String cmdUserName;
+    private String cmdPassword;
+
+    public DatabaseManager(String db, String userName, String password) {
+        this.cmdDB = db;
+        this.cmdUserName = userName;
+        this.cmdPassword = password;
+    }
+
+    public String checkCredentials() {
+        String resp = "";
+        if (!cmdDB.equals(db)) {
+            resp =  "db";
+        } else if (!cmdUserName.equals(userName)) {
+            resp = "username";
+        } else if (!cmdPassword.equals(password)) {
+            resp = "password";
+        }
+        return resp;
+    }
+
+    public void connection() {
+        try (Connection conn = DriverManager.getConnection(
+                "jdbc:postgresql://127.0.0.1:5432/" + cmdDB, cmdUserName, cmdPassword)) {
+            if (conn != null) {
+                System.out.println("Connected to the database!");
```

Comment:
Better to return the string and make 'sout' in the main class.

Reply:
+